worker/worker.go

```diff
@@ -678,15 +726,56 @@ func (w *worker) objectsKeyHandlerDELETE(jc jape.Context) {
 	jc.Check("couldn't delete object", w.bus.DeleteObject(jc.PathParam("key")))
 }
 
+func (w *worker) rhpRevisionsHandler(jc jape.Context) {
+	var req RHPContractsRequest
+	if jc.Decode(&req) != nil {
+		return
+	}
+
+	busContracts, err := w.bus.Contracts()
+	if jc.Check("failed to fetch contracts from bus", err) != nil {
+		return
+	}
+
+	cc := make([]contractCapability, len(busContracts))
```

Comment:
we definitely need a helper for this at some point, we have it 4 or 5 times already I think